web/interfaces.js

```diff
@@ -107,14 +107,20 @@ class IPDFLinkService {
  */
 class IPDFHistory {
   /**
-   * @param {Object} params
+   * @param {Object} options
+   * @param {*} options.fingerprint
+   * @param {boolean} options.resetHistory
+   * @param {boolean} options.updateUrl
```

Comment:
Also, both of these should be marked as optional (since they are).

Please see https://github.com/mozilla/pdf.js/blob/bf5a45ce6d0e9d070204859a39844bcdaf69e9aa/web/pdf_history.js#L36-L42 for the expected types (and what's optional).